src/nrnpython/nrnpy_nrn.cpp

```diff
@@ -1010,7 +1010,7 @@ static PyObject* nrnpy_set_psection_safe(PyObject* self, PyObject* args) {
 static PyObject* NPySecObj_psection(NPySecObj* self) {
     CHECK_SEC_INVALID(self->sec_);
     if (nrnpy_psection) {
-        auto arglist = nb::steal((PyObject*) Py_BuildValue("(O)", self));
+        auto arglist = nb::steal(Py_BuildValue("(O)", self));
```

Comment:
If you want these changes, put them with the other PR, it's still open and the diffs would just vanish.